vademecum_VT.md

```diff
@@ -43,8 +42,11 @@ Trova quella che fa più al caso tuo!
 Sono suddivise per genere, per consigliate, per più votate o per tendenza.
 
 #### Stylo |QRcode|
-Dalla versione Quantum, Firefox integra il nuovo sistema Style che velocizza il caricamente delle pagine web grazie all'ottimizzazione del foglio di stile (Stylo CSS).
-Questa tecnologia è parte del nuovo browser engine Servo ed è presente anche sulle versioni per dispositivi mobile, ovvero su Android e iOS.  
+Dalla versione Quantum, Firefox integra il nuovo sistema Stylo che velocizza il caricamente delle pagine web grazie all'ottimizzazione del foglio di stile (Stylo CSS) e al fatto che utilizza tutte le potenzialità dei sistemi multicore.
+Questa tecnologia è parte del nuovo Firefox Quantum desktop ed è presente anche su Android.
+
+#### Servo |QRcode|
+Servo è un browser sperimentale, scritto interamente _in Rust_, che serve per creare e testare i nuovi componenti del motore Gecko di Firefox Quantum, come il parser HTML o Stylo, il parser CSS.L'obiettivo è utlizzare totalmente le potenzialità di Rust migliorando la sicurezza, la modularità e le prestazioni, utilizzando i modermi processori  multicore e la GPU.
```

Comment:
browser engine* (non browser)
in **Rust*** (non _in Rust_)

Reply:
è meglio se revisiona @Mte90 per la validità del testo pere questa sezione

Reply:
Io, personalmente, lo modificherei in:

Servo è un browser engine sperimentale, scritto interamente in **Rust**.
In futuro, quando sarà stabile e completo, sostituirà l'ormai "anziano" Gecko. In termini di velocità, infatti, risulta essere molto più veloce di Gecko, grazie alle tante nuove tecnologie che si nascondono sotto un unico nome: **Servo**.
Alcune tecnologie sono implementate anche in Gecko (Stylo ne è un esempio eclatante).